src/vs/workbench/contrib/terminal/common/terminal.ts

```diff
@@ -24,6 +24,9 @@ export const KEYBINDING_CONTEXT_TERMINAL_SHELL_TYPE_KEY = 'terminalShellType';
 /** A context key that is set to the detected shell for the most recently active terminal, this is set to the last known value when no terminals exist. */
 export const KEYBINDING_CONTEXT_TERMINAL_SHELL_TYPE = new RawContextKey<string>(KEYBINDING_CONTEXT_TERMINAL_SHELL_TYPE_KEY, undefined, { type: 'string', description: nls.localize('terminalShellTypeContextKey', "The shell type of the active terminal") });
 
+export const KEYBINDING_CONTEXT_IPAD_KEY = 'isIPad';
+export const KEYBINDING_CONTEXT_IPAD = new RawContextKey(KEYBINDING_CONTEXT_IPAD_KEY, false, nls.localize(KEYBINDING_CONTEXT_IPAD_KEY, 'Whether the device is an ipad'));
+
```

Comment:
This won't work without actually hooking up setting the context key's value, I don't think we need it though since `if (isIPad)` handles it. If we wanted to make it easier to target iPad specifically we could add this to where the isWindows/isMac/etc. context keys live but it's probably not worth the effort right now.

Reply:
Yes I considered adding it to platform where those are. I do think it will work as a result of the isIpad check, right?

I considered setting it within terminalService or something, but thought that was unnecessary.

Reply:
I see what you mean actually